submissions/OlusujiBlessing/HTML/product.html

```diff
@@ -0,0 +1,71 @@
+<!Doctype html>
```

Comment:
It should be DOCTYPE and not Doctype. Also, remove the exclamation mark in the tag.